tests/e2e/Services/Tokens/TokensConsoleClientTest.php

```diff
@@ -107,6 +107,44 @@ public function testUpdateToken(array $data): array
         return $data;
     }
 
+    /**
+     * @depends testCreateToken
+     */
+    public function testExpiredTokenJWT(array $data): array
+    {
+        $fileId = $data['fileId'];
+        $bucketId = $data['bucketId'];
+
+        // Create a token with an expiry date in the past (expired)
+        $pastExpiry = DateTime::addSeconds(new \DateTime(), -3600); // 1 hour ago
+        $expiredToken = $this->client->call(Client::METHOD_POST, '/tokens/buckets/' . $bucketId . '/files/' . $fileId, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id']
+        ], $this->getHeaders()), [
+            'expire' => $pastExpiry,
+        ]);
+
+        $this->assertEquals(201, $expiredToken['headers']['status-code']);
```

Comment:
I would expect this to fail. We shouldn't be able to create a file token with an expiration in the past.

It might be best to just update the existing test to actually check the expiration.